From 6bcab3aad43a3977a793fe03ed5ba779195e690e Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 26 Nov 2013 13:47:41 +0000
Subject: [PATCH 0498/1077] extension Mantis 0029957: ConveyorComp: CUSS1.3
 Conveyor (without ConveyorAEA)

textidl.lib
V1.0.0.9
added new datastatus, eventcodes, statuscodes

code.idl.cmops.idl, characteristics.idl
V1.3.1
use CUSS1.3 IDL and XSD



git-svn-id: svn://localhost/SelfServiceCommon/trunk@618 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/cpp/textIDL/src/textcodes.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/textIDL/src/textcodes.cpp b/SelfServiceCommon/Massai/cpp/textIDL/src/textcodes.cpp
index a3c94440..5f2942eb 100644
--- a/SelfServiceCommon/Massai/cpp/textIDL/src/textcodes.cpp
+++ b/SelfServiceCommon/Massai/cpp/textIDL/src/textcodes.cpp
@@ -198,8 +198,8 @@ namespace MASSAI
         case STATE_EXPLANATION:               return "STATE_EXPLANATION";
         case ACTIVE_TRANSFER:                 return "ACTIVE_TRANSFER";
         case TRANSACTION_EXPLANATION:         return "TRANSACTION_EXPLANATION";
-        case VERSION_EXPLANATION              return "VERSION_EXPLANATION";
-        case UPDATE_REQUEST                   return "UPDATE_REQUEST";
+        case VERSION_EXPLANATION:             return "VERSION_EXPLANATION";
+        case UPDATE_REQUEST:                  return "UPDATE_REQUEST";
 
         case RC_REFERENCE:                    return "REFERENCE";
         case RC_STATE:                        return "STATE";
-- 
2.41.0.windows.1

